### PR TITLE
feat(container): use `@container` query for breakpoint by default

### LINKED ISSUE #264: Add a `config` to use `@container` with breakpoint
### Subject

`config`

### Description

We should make breakpoints in component preview use `@container` media query.

### Are you willing to participate in the development of this feature and create a pull request?

None

### PR DESCRIPTION
**Pull request by taroj1205**
Closes #264

## Description

I have made component examples use `@container` query for breakpoint by default.

## Current behavior (updates)

Using breakpoint values were using `@media screen` query.

## New behavior

Now uses `@container`.

## Is this a breaking change (Yes/No):

No

## Additional Information

`config.ts` does not have an effect on Windows, but I will fix this in another PR.

### CODE REVIEW

File: components/data-display/component-preview.tsx [hunk 1]
```diff
@@ -69,6 +70,13 @@ export const ComponentPreview = memo(
         return props
       }, [_containerProps])
 
+      const customConfig = extendConfig({
```
Comment (nakayan5) [inline components/data-display/component-preview.tsx:73]: @taroj1205 
[nits]
I think this process should be done in useAsync.
If you do that, you won't need to change line 88.

Comment (taroj1205) [reply components/data-display/component-preview.tsx:73]: You're right, I'll fix it.

Comment (nakayan5) [reply components/data-display/component-preview.tsx]: @taroj1205 
Why did you stop using `extendConfig` ?
(You don't have to use it.)

Comment (taroj1205) [reply components/data-display/component-preview.tsx]: @nakayan5 Thanks, I also think I should use `extendConfig`. I will fix it now.

**Review by nakayan5 — APPROVED** [review]

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**